dashboard/app/views/layouts/_footer.html.haml

```diff
@@ -38,6 +38,6 @@
         -# user input or to persist data without also updating it to enforce UTF-8
         = form_tag(locale_url, method: :post, id: 'localeForm', style: 'margin-bottom: 0px;', enforce_utf8: false) do
           = hidden_field_tag :user_return_to, request.url
-          = select_tag :locale, options_for_select(options_for_locale_select, locale), onchange: 'this.form.submit();'
+          = select_tag :locale, options_for_select(options_for_locale_select, locale), { onchange: 'this.form.submit();', 'aria-label':  I18n.t('footer_locale_select') }
```

Comment:
The rest of this file appears to just use `t()`, so this should probably remain consistent.

Reply:
And should be a `footer.` style reference to a string near [here](https://github.com/code-dot-org/code-dot-org/blob/407599dea267d83c2fcbd2ae5e90244f0990d73c/dashboard/config/locales/en.yml#L600).

Reply:
I noticed that based on your first comment. Now I'm a little confused though, does editing [this Google sheet](https://docs.google.com/spreadsheets/d/1Tq7VqZALgRA0wYk0HDfEOTyRI0TM2Dir2rloXIPGCgU/edit#gid=0) no longer apply? Or does the `dashboard` still source information from there, and there's just some massaging that happens in between?

Reply:
Seems like these get added manually? https://github.com/code-dot-org/code-dot-org/commits/staging/dashboard/config/locales/en.yml

Reply:
That google sheet is just for pegasus code.  For dashboard, you can add the new string to `dashboard/config/locales/en.yml` in this PR.